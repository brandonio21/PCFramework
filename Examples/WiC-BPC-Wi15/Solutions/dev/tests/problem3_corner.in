Blue Silver Gold Blue Green
Blue Green Red
Pink Magenta Cyan Red Blue Green Yellow Orange Purple White Black
Pink Magenta Cyan Red Pink Blue Green Blue Yellow Orange White Purple White Black Pink Pink Pink

Blue Blue Blue Blue Blue
Blue
Hadurp flerp merp clarghleflargle poppy chicken
Food
chicken nuggets of death
flee floo floe flee floo flop flop
moo cow
UNICORN UNICORN UNICORN DINO UNICORN UNICORN UNICORN
Blue blue BLUE blUe BLuE
pink blue green PINK BLUE green
pink nipk knip nkpi
pinkk pinnk piink ppink
