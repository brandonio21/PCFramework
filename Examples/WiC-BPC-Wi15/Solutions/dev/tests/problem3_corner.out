4
3
11
11
0
1
1
6
1
4
4
2
2
1
3
4
4
